rapidsms_multimodem/views.py

```diff
@@ -70,9 +70,20 @@ def receive_multimodem_message(request, server_slug):
         from_number = message.find('ModemNumber').text
 
         # ModemNumber is simply 1 for single-port modems and it's a string of
-        # 'port_numer:phone_number' for multiport modems.
+        # 'port_number:phone_number' for multiport modems.
+
+        # RR: This gets a little more complicated than that. If the modem is not set up correctly,
+        # this field will be only the index of the number. The possible values are:
+        # <ModemNumber>[Index]:[Configured Phone Number]</ModemNumber> -- for multi-port and configured
+        # <ModemNumber>[Configured Phone Number]</ModemNumber> -- for single port and configured
```

Comment:
So this part is unfortunately not quite right. Single Port Modems only put their modemnumber in there, no phone number. The only valid ModemNumber sent by our 1-port modem is: `<ModemNumber>1</ModemNumber>`.

Given that, I don't think there's a good way of differentiating a properly configured single-port modem from an improperly-configured multiport modem, aside from adding a setting to the configuration (yuck...)